src/main/twirl/scalajs/jsdeps.scala.html

```diff
@@ -1,3 +1,5 @@
+@import play.twirl.api.TwirlFeatureImports.defining
```

Comment:
The project compiles fine without the import. What is the reason for adding this import?